src/sentry/api/endpoints/organization_incident_comment_index.py

```diff
@@ -23,7 +23,7 @@ def post(self, request, organization, incident):
                 incident,
                 IncidentActivityType.COMMENT,
                 # XXX: Serialization fails if user is a `SimpleLazyObject`
-                user=request.user._wrapped,
+                user=getattr(request.user, '_wrapped', request.user),
```

Comment:
How do we handle impersonation in other parts of the application? I don't see any other usage like this.

Reply:
It's because I'm using `_wrapped`. Basically I needed to do this because `SimpleLazyObject` ends up assigned against the activity, and it fails when we try to serialize, because it tries to look up `SimpleLazyObject` in the registry and can't find it.

Actually, maybe I can fix this by specifying the serializer rather than relying on the registry, might try that.